private_set_intersection/deps.bzl

```diff
@@ -89,6 +89,10 @@ def psi_deps():
             ],
         )
 
+    # gRPC for PJC.
+    grpc_deps()
+    apple_rules_dependencies()
```

Comment:
Instead of handling specific cases, doesn't this work? https://github.com/grpc/grpc/pull/20586#issuecomment-546182904

I see that PJC does something similar https://github.com/google/private-join-and-compute/blob/master/WORKSPACE#L79

Reply:
This only works inside a WORKSPACE file, because `load` is only allowed at the beginning of `.bzl` files.
Putting the contents of preload.bzl and deps.bzl into WORKSPACE on the other hand has the disadvantage that it makes it harder for others to use the library.